app/javascript/providers/StudySearchProvider.js

```diff
@@ -104,7 +104,7 @@ export function PropsStudySearchProvider(props) {
 
     setSearchState({
       params,
-      isError: false,
+      isError: results.ok? !results.ok: false,
```

Comment:
Relatedly:

A component-level test that verifies the "Sorry, an error has occurred" error message appears in the rendered ResultsPanel UI when `isError` is `true` would prevent this bug from (re-)emerging.

I don't consider such a new test blocking, but we should at least document it as a "TODO" or "To consider" in the code.